From 3473167366fc21a2d26ca9c1baa727a773627acb Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Sun, 2 Mar 2025 08:00:54 +0100
Subject: [PATCH 5/7] Revert "components/linux.bst: add option needed for HDMI
 output on the Rock 5B"

This reverts commit 15b9d31b2cc9bab3d7046ef9f8eaa44c49d02252.
---
 files/linux/fdsdk-config.sh | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index 3dc8dd3d8..df64d67b7 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -629,14 +629,6 @@ case "$arch" in
     ;;
 esac
 
-# Needed for HDMI display on Rock 5B
-case "$arch" in
-    aarch64)
-        module PHY_ROCKCHIP_SAMSUNG_HDPTX
-        enable ROCKCHIP_DW_HDMI_QP
-    ;;
-esac
-
 # DRM for virtual machines
 enable DRM_VIRTIO_GPU
 case "$arch" in
-- 
2.47.2

